### ISSUE TITLE
TeenPies list index out of range

### ISSUE THREAD

**Issue by d2dyno** [PAhelper/PhoenixAdult.bundle#271]
```
2019-07-11 00:51:55,719 (146cebdfe700) :  INFO (__init__:46) - *******MEDIA TITLE****** Teenpies Claire Black
2019-07-11 00:51:55,720 (146cebdfe700) :  INFO (__init__:53) - Getting Search Settings for: Teenpies Claire Black
2019-07-11 00:51:55,720 (146cebdfe700) :  INFO (PAsearchSites:882) - mediaTitle w/ possible abbrieviation: Teenpies Claire Black
2019-07-11 00:51:55,727 (146cebdfe700) :  INFO (PAsearchSites:1035) - mediaTitle w/ possible abbrieviation fixed: Teenpies Claire Black
2019-07-11 00:51:55,728 (146cebdfe700) :  INFO (PAsearchSites:1050) - ^^^^^^^ siteID: 189
2019-07-11 00:51:55,728 (146cebdfe700) :  INFO (PAsearchSites:1052) - ^^^^^^^ Shortening Title
2019-07-11 00:51:55,728 (146cebdfe700) :  INFO (PAsearchSites:1053) - teenpies  vs teen pies
2019-07-11 00:51:55,728 (146cebdfe700) :  INFO (PAsearchSites:1101) - 2
2019-07-11 00:51:55,729 (146cebdfe700) :  INFO (PAsearchSites:1101) - 3
2019-07-11 00:51:55,729 (146cebdfe700) :  INFO (PAsearchSites:1101) - 6
2019-07-11 00:51:55,729 (146cebdfe700) :  INFO (PAsearchSites:1101) - searchTitle (before date processing): Claire Black
2019-07-11 00:51:55,729 (146cebdfe700) :  INFO (__init__:63) - Search Title: Claire Black
2019-07-11 00:51:55,729 (146cebdfe700) :  INFO (__init__:72) - Claire%20Black
2019-07-11 00:51:55,731 (146cebdfe700) :  DEBUG (networking:161) - Fetching 'https://www.teamskeet.com/t1/search/results/?query=Claire%20Black' from the HTTP cache
2019-07-11 00:51:55,737 (146cebdfe700) :  DEBUG (networking:161) - Fetching 'https://www.teamskeet.com/t1/trailer/view/eavesdropping_and_pussy_popping/claire_black?thumb=04.jpg' from the HTTP cache
2019-07-11 00:51:55,745 (146cebdfe700) :  CRITICAL (agentkit:1014) - Exception in the search function of agent named 'PhoenixAdult', called with keyword arguments {'year': None, 'id': '94291', 'name': 'Teenpies Claire Black'} (most recent call last):
  File "/usr/lib/plexmediaserver/Resources/Plug-ins-4b7ace214/Framework.bundle/Contents/Resources/Versions/2/Python/Framework/api/agentkit.py", line 1007, in _search
    agent.search(*f_args, **f_kwargs)
  File "/config/Library/Application Support/Plex Media Server/Plug-ins/PhoenixAdult.bundle/Contents/Code/__init__.py", line 378, in search
    results = PAsearchSites.siteTeamSkeet.search(results,encodedTitle,title,searchTitle,siteNum,lang,searchByDateActor,searchDate, searchSiteID)
  File "/config/Library/Application Support/Plex Media Server/Plug-ins/PhoenixAdult.bundle/Contents/Code/siteTeamSkeet.py", line 11, in search
    titleNoFormatting = scenePage.xpath('//title')[0].text_content().split(" | ")[1]
  File "/usr/lib/plexmediaserver/Resources/Plug-ins-4b7ace214/Framework.bundle/Contents/Resources/Versions/2/Python/Framework/code/sandbox.py", line 108, in <lambda>
    _getitem_           = lambda x, y: x.__getitem__(y),
IndexError: list index out of range
```

**Comment by d2dyno** [comment]
This actually looks to be an issue when loading the scene on their site period, not the agent:
https://www.teamskeet.com/t1/trailer/view/eavesdropping_and_pussy_popping/claire_black

**Comment by claygoldfinch** [comment]
Resolved in PR #264.
